### ISSUE TITLE
Update team guidance related to code ownership

### ISSUE THREAD

**Issue by little-oddball** [department-of-veterans-affairs/va.gov-team#43415]

**Comment by little-oddball** [comment]
In talking w/ @mchelen some initial thoughts were if the situation occurs where there is orphaned / no long supported code that is causing issues with the Platform, then tickets should be generated to track and resolve by removing the offending code if not addressed after a defined amount of time. 

The key point is that Platform teams are not responsible for maintaining code added by VFS teams and guidance needs to be updated to make that clear.

**Comment by timwright12** [comment]
> The key point is that Platform teams are not responsible for maintaining code added by VFS teams and guidance needs to be updated to make that clear.

I agree with this, but at the same time we definitely can't go around removing features from the site because they're in contract limbo - that would be like half the site, and extremely high effort taxing already maxed out teams.

Code ownership in vets-website currently defaults to @department-of-veterans-affairs/platform-release-tools  - we don't update the code or add to it in anyway (no team = no roadmap = no features to add) unless we have to in order to achieve a platform objective (upgrading react, for instance), but if there were a critical feature that was causing issue we would certainly take a pass at fixing it to maintain a consistent veteran experience.

Would love to be in these chats if possible.

**Comment by annekerr49** [comment]
Is there any update of the progress of this effort?
